pom.xml

```diff
@@ -2349,8 +2439,11 @@
                             <user.language>en</user.language>
                             <user.region>US</user.region>
                             <geomesa.scan.ranges.target>500</geomesa.scan.ranges.target>
-                            <geomesa.hbase.test.servers>2</geomesa.hbase.test.servers>
+                            <geomesa.accumulo.test.tablet.servers>1</geomesa.accumulo.test.tablet.servers>
+                            <geomesa.hbase.test.region.servers>1</geomesa.hbase.test.region.servers>
```

Comment:
Ok, somewhere else one of the defaults is set to 2.  Thoughts?

Reply:
I think 2 is the default used by hbase/accumulo, which is why it's 2 in the code but 1 here. if that doesn't make sense i don't mind updating it